code.js

```diff
@@ -0,0 +1,53 @@
+//Kruskal's Algorithm
+function kruskal(graph) {
+
+    //Start with an empty tree T
+    var tree = [];
+    for (let i = 0; i < graph.length; i++) {
+        tree[i] = [];
+    }
+
+    //Add the minimum weight edge to T unless it forms a cycle
+    while (graph.flat().length > 0) {
+        let edge, min = Infinity;
+        for (let i = 0; i < graph.length; i++) {
+            for (let j = 0; j < graph[i].length; j++) {
+                if (graph[i][j][1] < min) {
+                    min = graph[i][j][1];
```

Comment:
Feels like this could be more efficient, e.g. with a priority queue as we talked about in class.